### ISSUE TITLE
Kill download job

### ISSUE THREAD

**Issue by painter1** [Prodiguer/synda#75]
Right now, the only clean way to stop a download in progress is with "synda remove...".  But this also removes all the files which have been downloaded.  Sometimes you want to stop a download job and keep the data which has already come.

**Comment by painter1** [comment]
An example is if the GridFTP server fails and you want to switch to another GridFTP node, or use HTTP.

**Comment by glevava** [comment]
`synda remove` doesn't really "stop" the download. It purges the involved datasets from the database and remove the corresponding data on the filesystem.
To stop the download process, the only way is to stop the service/daemon : `service sdt stop`.
In addition, `synda remove` is able to delete data at the "dataset" granularity using `synda remove <dataset_id>`.
In the case of a fallback on HTTP protocol when GridFTP downloads failed you can add `http_fallback=true` in your sdt.conf.

**Comment by painter1** [comment]
Yes, the real problem is that you can't stop a download without purging the database and data.  In production use, there will be several simultaneous download jobs, from several nodes.  So stopping the daemon isn't a practical option.
Thanks for telling me about the `http_fallback` setting.  Does that use the same server as for GridFTP?  Sometimes they will be different.

**Comment by painter1** [comment]
I just tried stopping the daemon.  When I restarted the daemon,  the download started again.  Is there any way to kill the download without losing what's in the database and file system?

**Comment by painter1** [comment]
"synda reset" will stop all downloads without purging the database and data.  This is enough to get by on, but it would be very helpful if it could be restricted to data defined by a particular selection file.
